### PR TITLE
[cli] Require --force with account:claim-attestation-service-url for non-TLS urls

### PR DESCRIPTION
**Pull request by timmoreton**
### Description

In the future we will be requiring TLS for Attestation Service URLs. This adds a check to the CLI command to require `--force` to add a URL that doesn't begin `https://`.
 
### Tested

Locally

### Backwards compatibility

In the unlikely event that users have scripted this, and do not have TLS deployed, scripts will need modification.

### CODE REVIEW

**Review by nambrot — APPROVED** [review]

**Comment by medhakothari** [comment]
Hi @timmoreton! Could you update the [changelog](https://github.com/celo-org/celo-monorepo/blob/master/packages/cli/CHANGELOG.md) in this PR as well?

File: packages/cli/src/commands/account/claim-attestation-service-url.ts [hunk 1]
```diff
@@ -8,17 +9,24 @@ export default class ClaimAttestationServiceUrl extends ClaimCommand {
     ...ClaimCommand.flags,
     url: Flags.url({
       required: true,
-      description: 'The url you want to claim',
+      description: 'The URL you want to claim. Should begin http://',
```
Comment (aslawson) [inline packages/cli/src/commands/account/claim-attestation-service-url.ts:12]: you mean `https://`?

Comment (aslawson) [inline packages/cli/src/commands/account/claim-attestation-service-url.ts:12]:
```suggestion
      description: 'The URL you want to claim. Should begin https://',
```

File: packages/docs/command-line-interface/account.md [hunk 2]
```diff
@@ -147,12 +149,12 @@ OPTIONS
                                                      index addresses for local signing. Example --ledgerCustomAddresses
                                                      "[4,99]"
 
-  --url=https://www.celo.org                         (required) The url you want to claim
+  --url=https://www.celo.org                         (required) The URL you want to claim. Should begin http://
```
Comment (aslawson) [inline packages/docs/command-line-interface/account.md:152]:
```suggestion
  --url=https://www.celo.org                         (required) The URL you want to claim. Should begin https://
```

**Pull request merged**